docs/resources/flexible_ip.md

```diff
@@ -0,0 +1,67 @@
+---
+page_title: "Scaleway: scaleway_flexible_ip"
+description: |-
+  Manages Scaleway Flexible IPs.
+---
+
+# scaleway_flexible_ip
+
+Creates and manages Scaleway flexible IPs.
+For more information, see [the documentation](https://developers.scaleway.com/en/products/flexible-ip/api).
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_flexible_ip" "main" {
+    reverse = "my-reverse.com"
+}
+```
+
+### With baremetal server
+
+```hcl
+data "scaleway_account_ssh_key" "main" {
+    name = "main"
+}
+
+resource "scaleway_baremetal_server" "base" {
+    zone = "fr-par-2"
+    offer = "EM-A210R-HDD"
+    os = "96e5f0f2-d216-4de2-8a15-68730d877885"
```

Comment:
Could you use: https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/data-sources/baremetal_os